.travis.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       script:
         - yarn lint:js
         - yarn test
-        - yarn test:node
+        - npm run test:node
```

Comment:
Why change this to npm?

Reply:
This is a known issue https://github.com/tomdale/ember-cli-addon-tests/issues/198, should be fixed by a WIP PR https://github.com/tomdale/ember-cli-addon-tests/pull/105